tests/BenchmarkDotNet.Tests/Exporters/ApprovedFiles/ApprovalTest.Exporter.MarkdownExporter-stackoverflow_Russian (Russia).approved.txt

```diff
@@ -0,0 +1,13 @@
+﻿
```

Comment:
It also would be great to rename `Russian (Russia)` to `ru-ru` and `English (United States)` to `en-us`.